File: src/com/androhi/androiddrawableviewer/action/EditTargetResDirAction.java
package com.androhi.androiddrawableviewer.action;

import com.androhi.androiddrawableviewer.form.SettingsDialog;
import com.intellij.icons.AllIcons;
import com.intellij.openapi.actionSystem.AnAction;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.intellij.openapi.project.Project;

public class EditTargetResDirAction extends AnAction {

    public EditTargetResDirAction() {
        super("Edit directory", "Edit target resource directory", AllIcons.General.Settings);
    }

    @Override
    public void actionPerformed(AnActionEvent anActionEvent) {
        Project project = anActionEvent.getProject();
        SettingsDialog dialog = new SettingsDialog(project);
        dialog.show();
    }

    @Override
    public void update(AnActionEvent e) {
        super.update(e);
    }
}

File: src/main/kotlin/com/androhi/androiddrawableviewer/action/EditTargetResDirAction.kt
package com.androhi.androiddrawableviewer.action

import com.androhi.androiddrawableviewer.form.SettingsDialog
import com.intellij.icons.AllIcons
import com.intellij.openapi.actionSystem.AnAction
import com.intellij.openapi.actionSystem.AnActionEvent

class EditTargetResDirAction : AnAction("Edit directory", "Edit target resource directory", AllIcons.General.Settings) {

    override fun actionPerformed(anActionEvent: AnActionEvent?) {
        val project = anActionEvent?.project
        project?.let {
            SettingsDialog(it).show()
        }
    }

    override fun update(e: AnActionEvent?) {
        super.update(e)
    }
}